libs/ledger-live-common/src/exchange/swap/utils/index.ts

```diff
@@ -172,6 +172,11 @@ export const shouldShowKYCBanner = ({
   return kycStatus !== KYC_STATUS.approved;
 };
 
+export const registrationIsRequired = (provider: string): boolean => {
```

Comment:
```suggestion
export const isRegistrationRequired = (provider: string): boolean => {
```

For better naming convention, name Functions as verbs.
cf. https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0

Reply:
Done